netbout-web/src/main/java/com/netbout/rest/Markdown.java

```diff
@@ -58,7 +58,7 @@
          * Ctor.
          */
         public Default() {
-            this.processor = new MarkdownPegdown();
+            this.processor = new MarkdownTxtmark();
```

Comment:
@dmzaytsev can we create puzzle for it? I'm not sure if we should leave dead code in the project. It always could be restored from VCS if needed.